### ISSUE TITLE
Cmake compilation error

### ISSUE THREAD

**Issue by DevFlagship** [BelledonneCommunications/linphone-android#384]
I try command "make", but get this error:
`> Task :compileDebugJavaWithJavac FAILED
/home/devflagship/linphone-android/src/android/org/linphone/chat/GroupChatFragment.java:611: error: cannot find symbol
            mChatRoomSecurityLevel.setVisibility(View.GONE);
            ^
  symbol:   variable mChatRoomSecurityLevel
  location: class GroupChatFragment
`
How i can fix this error? 
my previous command was `./prepare.py` without any features

**Comment by cnscns** [comment]
You should download the 4.0.1 version and replace the src directory by using src of the 4.0.1

**Comment by behnamriahi** [comment]
> You should download the 4.0.1 version and replace the src directory by using src of the 4.0.1

what? git clone not is latest version?!

**Comment by DevFlagship** [comment]
No, git clone is latest version, but this project have many branches, and master is development branch. release version for compile in the end of branches list

**Comment by behnamriahi** [comment]
> No, git clone is latest version, but this project have many branches, and master is development branch. release version for compile in the end of branches list

how can get 4.0.1 version?

my command is:
`git clone git://git.linphone.org/linphone-android.git --recursive`
`./prepare.py`
`make`

and error :
`> Task :compileDebugJavaWithJavac FAILED
/home/user/Downloads/linphone-android/src/android/org/linphone/chat/GroupChatFragment.java:611: error: cannot find symbol
            mChatRoomSecurityLevel.setVisibility(View.GONE);`
